scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -22,14 +22,14 @@ type LaneOptions = {
 
 export class LaneListCmd implements Command {
   name = 'list';
-  description = `list lanes`;
+  description = `list locally imported/created lanes`;
```

Comment:
why not just "list local lanes"?

Reply:
wanted to be more verbose about what that means
But can see that local lanes is analogous enough to local branches that people should intuitively know what local lanes means
Changing